### ISSUE TITLE
Doxygen comment suggestion in help collides with clang-format

### ISSUE THREAD

**Issue by JohnC32** [doxygen/doxygen#8591]
In https://www.doxygen.nl/manual/docblocks.html we have:

     Some people like to make their comment blocks more visible in the documentation. For this purpose you can use the following:

     /********************************************//**
      *  ... text
      ***********************************************/
     (note the 2 slashes to end the normal comment block and start a special comment block).

Writing comments like this collide with clang-format, thus can this suggestion be removed or add a note that they should not  be used if using clang-format. 

If you use clang-format, a comment of the form:

    /**************//**
     * Doxygen comment
     *****************/

will be reflowed by clang-format to become

    /**************/ /**
                      * Doxygen comment
                      *****************/

and if you set the "ColumnLimit" in clang-format, then the comment becomes very long and narrow and odd looking.

**Comment by JohnC32** [comment]
Note, I looked into the clang-format source and reason for this behavior is clang-format will align comments to the statement to the left, e.g.

     int foo = 1; /* a comment that spans a few lines describing what the statement to the left of
                   * the comment is doing */

and the /*******/ is just treated as a statement.

**Comment by albert-github** [comment]
I read in the clang documentation (https://clang.llvm.org/docs/ClangFormatStyleOptions.html):
> When using clang-format command line utility or clang::format::reformat(...) functions from code, one can either use one of the predefined styles (LLVM, Google, Chromium, Mozilla, WebKit, Microsoft) or create a custom style by configuring specific style options.

- Is it possible to create a custom style to overturn this clang behavior?

Notwithstanding the above a small message in the documentation might be useful.

**Comment by albert-github** [comment]
@JohnC32 

Do you know how whether of not clang-format reformat will "maltreat"  the following?
```
/////////////////////////////////////////////////
/// ... text ...
/////////////////////////////////////////////////
```

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8593 enhancing the documentation with a general note.

**Comment by JohnC32** [comment]
Hi 

clang-format will not 'reflow' the following

     /////////////////////////////////////////////////
     /// ... text ...
     /////////////////////////////////////////////////

I've been using clang-format for a while on a large code base and am fairly familiar with the options. I could not find a way to create a custom option to tell it to not reflow the /******//** style comment. I did spend time looking at the clang-format code and think I understand why it's reflowing these comments. clang-format is quite popular and so is Doxygen. I suspect very few people are using the /*****//** style comment with clang-format. Within my company, I found only a few cases in a massive code base.

**Comment by JohnC32** [comment]
> I've just pushed a proposed patch, pull request #8593 enhancing the documentation with a general note.

This looks good. Thank you.

**Comment by albert-github** [comment]
Please don't close the issue this will be done, semi-automatically, when the proposed patch is integrated into the master version and subsequently into a release.

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
